Source/Tools/FEXLoader/WindowsDummyHandlers.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include <FEXCore/Core/SignalDelegator.h>
+#include <FEXCore/HLE/SyscallHandler.h>
+
+#ifdef _WIN32
+namespace FEX::WindowsHandlers {
+
+class DummySyscallHandler: public FEXCore::HLE::SyscallHandler, public FEXCore::Allocator::FEXAllocOperators {
+  public:
+  uint64_t HandleSyscall(FEXCore::Core::CpuStateFrame *Frame, FEXCore::HLE::SyscallArguments *Args) override {
+    // Don't do anything
+    return 0;
+  }
+
+  FEXCore::HLE::SyscallABI GetSyscallABI(uint64_t Syscall) override {
+    // Don't do anything
+    return {0, false, 0};
+  }
+
+  // These are no-ops implementations of the SyscallHandler API
+  std::shared_mutex StubMutex;
+  FEXCore::HLE::AOTIRCacheEntryLookupResult LookupAOTIRCacheEntry(FEXCore::Core::InternalThreadState *Thread, uint64_t GuestAddr) override {
```

Comment:
You can drop this mutex now the LookupAOTIRCacheEntry return type has changed

Reply:
Indeed. Removed.